db/models/orders.js

```diff
@@ -0,0 +1,61 @@
+const client = require("./client");
+
+const getAllOrders = async () => {
+  try {
+    const { rows: orders } = await client.query(`
+      SELECT routines.*, users.username AS "creatorName"
```

Comment:
change routines.* to orders.*

Reply:
GOOOD catch! I had already done that, but then reverted back for another reason and forgot to change it again.